sdks/python/src/opik/api_objects/dataset/dataset.py

```diff
@@ -205,6 +205,32 @@ def insert_from_json(
 
         self.insert(new_items)
 
+    def read_json_from_file(
+        self,
+        file_path: str,
+        keys_mapping: Optional[Dict[str, str]] = None,
+        ignore_keys: Optional[List[str]] = None,
+    ) -> None:
+        """
+        Read JSONL from a file and insert it into the dataset.
+
+        Args:
+            file_path: Path to the JSONL file
+            keys_mapping: dictionary that maps json keys to item fields names
+                Example: {'Expected output': 'expected_output'}
+            ignore_keys: if your json dicts contain keys that are not needed for DatasetItem
+                construction - pass them as ignore_keys argument
+        """
+        items = []
```

Comment:
This logic for creating the list of dataset items should be moved to `converters.from_jsonl_file` (similarly as for the other `from_*` functions in this module).